.github/workflows/publish_wiki.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish Wiki
+
+on:
+  push:
+    paths:
+      - .github/wiki/**
+    branches:
+      - develop
+      - chore/add-template-development-docs
```

Comment:
Should we remove this 👍

Reply:
yup, removed in https://github.com/nimblehq/elixir-templates/pull/135/commits/f9baecf8cf1d6d6b67df71b886d10204d2aa354c